.eslintrc.js

```diff
@@ -0,0 +1,107 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+  },
+  extends: [
+    'plugin:react/recommended',
+    'airbnb-typescript',
+    'plugin:import/typescript',
+    // @NOTE: Make sure this is always the last element in the array.
+    'plugin:prettier/recommended',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaVersion: 2020,
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+    project: './tsconfig.json',
+  },
+  plugins: [
+    'react',
+    'react-hooks',
+    'jsx-a11y',
+    'import',
+    'prettier',
+    '@typescript-eslint',
+  ],
+  settings: {
+    react: {
+      pragma: 'React', // Pragma to use, default to "React"
+      fragment: 'Fragment', // Fragment to use (may be a property of <pragma>), default to "Fragment"
+      version: 'detect', // React version. "detect" automatically picks the version you have installed.
+      // You can also use `16.0`, `16.3`, etc, if you want to override the detected value.
+      // default to latest and warns if missing
+      // It will default to "detect" in the future
+    },
+    'import/parsers': {
+      '@typescript-eslint/parser': ['.ts', '.tsx'],
+    },
+    'import/resolver': {
+      typescript: {},
+    },
+    'import/no-unresolved': [
+      2,
+      { ignore: ['^@theme', '^@docusaurus', '^@site'] },
+    ],
+  },
+  rules: {
+    'import/prefer-default-export': 0,
+    'import/no-unresolved': 'error',
```

Comment:
Ignore stuff should be in `rules` instead.
```suggestion
  },
  rules: {
    'import/prefer-default-export': 0,
    'import/no-unresolved': [
      'error',
      { ignore: ['^@theme', '^@docusaurus', '^@site'] },
    ],
```

Reply:
https://github.com/immutable/imx-docs/pull/55/commits/4ed57e7c1ae58dede89be03e157f96216f038adb